src/module/removeAll.js

```diff
@@ -0,0 +1,9 @@
+const removeAll = (localValue, displayTask) => {
+  const storage = [...localValue];
+  const remainingValue = storage.filter((item) => item.completed !== true);
```

Comment:
- [ ] Good job adding A function for the "Clear all completed" but since the task is not getting added I  was unable to try this functionality. Please make sure on the button click, all completed tasks are cleared.